### ISSUE TITLE
Sermanet baseline

### ISSUE THREAD

**Issue by cdiwork** [cdiwork/gtsrb#1]
A few architectures of the Sermanet et al. paper and a minimal amount of tuning of parameters. The best one is in line with the results in the paper ~97% accuracy.

The base architecture is 

nn.Sequential {
  [input -> (1) -> (2) -> (3) -> (4) -> (5) -> (6) -> (7) -> (8) -> (9) -> (10) -> (11) -> (12) -> (13) -> (14) -> (15) -> (16) -> (17) -> (18) -> (19) -> output](1): nn.SpatialContrastiveNormalization
  (2): nn.SpatialConvolutionMM(in: 1, out: 100, kW: 5, kH: 5)
  (3): nn.Tanh
  (4): nn.ReLU
  (5): nn.SpatialSubtractiveNormalization
  (6): nn.SpatialDivisiveNormalization
  (7): nn.SpatialMaxPooling(kW: 2, kH: 2, dW: 2, dH: 2)
  (8): nn.SpatialConvolutionMM(in: 100, out: 200, kW: 5, kH: 5)
  (9): nn.Tanh
  (10): nn.ReLU
  (11): nn.SpatialSubtractiveNormalization
  (12): nn.SpatialDivisiveNormalization
  (13): nn.SpatialMaxPooling(kW: 2, kH: 2, dW: 2, dH: 2)
  (14): nn.Reshape(5000)
  (15): nn.Linear(5000 -> 100)
  (16): nn.Tanh
  (17): nn.Linear(100 -> 43)
  (18): nn.Tanh
  (19): nn.Reshape(43)
}   

The MS architecture is 
nn.Sequential {
  [input -> (1) -> (2) -> (3) -> (4) -> (5) -> (6) -> (7) -> (8) -> (9) -> (10) -> (11) -> (12) -> (13) -> (14) -> (15) -> output](1): nn.SpatialConvolutionMM(in: 1, out: 100, kW: 5, kH: 5)
  (2): nn.Tanh
  (3): nn.ReLU
  (4): nn.SpatialSubtractiveNormalization
  (5): nn.SpatialDivisiveNormalization
  (6): nn.SpatialMaxPooling(kW: 2, kH: 2, dW: 2, dH: 2)
  (7): nn.Replicate
  (8): nn.SplitTable
  (9): nn.ParallelTable {
    input
      |`-> (1): nn.Sequential {
      |      [input -> (1) -> (2) -> (3) -> (4) -> output]
      |      (1): nn.SpatialConvolutionMM(in: 100, out: 200, kW: 5, kH: 5)
      |      (2): nn.ReLU
      |      (3): nn.SpatialMaxPooling(kW: 2, kH: 2, dW: 2, dH: 2)
      |      (4): nn.Reshape(5000)
      |    }
      |`-> (2): nn.Sequential {
      |      [input -> (1) -> (2) -> output]
      |      (1): nn.SpatialMaxPooling(kW: 2, kH: 2, dW: 2, dH: 2)
      |      (2): nn.Reshape(4900)
      |    }
       ... -> output
  }
  (10): nn.JoinTable
  (11): nn.Linear(9900 -> 100)
  (12): nn.Tanh
  (13): nn.Linear(100 -> 43)
  (14): nn.Tanh
  (15): nn.Reshape(43)
}
